app/src/test/java/com/globant/counter/android/mvp/presenter/PresenterTest.java

```diff
@@ -0,0 +1,59 @@
+package com.globant.counter.android.mvp.presenter;
+
+import com.globant.counter.android.mvp.model.CountModel;
+import com.globant.counter.android.mvp.view.CountView;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class PresenterTest {
+
+    private CountPresenter presenter;
+    private CountModel model;
+    private CountView view;
```

Comment:
In the presenter test. Always View and Model are mocks.

Replace with
`@Mock CountModel model;
 @Mock CountView view;`